src/flatpakUtils.ts

```diff
@@ -5,8 +5,8 @@ export interface Runtime {
     version: string
```

Comment:
could also rename the interface to something more generic for both app and runtime. Maybe `FlatpakEntry`?

Reply:
sounds good to me